spec/requests/courses_spec.rb

```diff
@@ -109,11 +104,18 @@
 
         it "returns the Courses that the student is enrolled in" do
           get "/api/courses/visible"
-          expect(response).to have_http_status(200)
+          expect(response).to be_ok
           expect(json_response.size).to eq(1)
           expect(json_response.first).to include(course_properties)
         end
       end
     end
+
+    describe "GET /api/courses/:id/students" do
+      it "returns forbidden" do
```

Comment:
To line up with the other "forbidden" examples, let's rename this to `it "is inaccessible"`.